usr/local/share/bastille/create.sh

```diff
@@ -372,9 +372,9 @@ help|-h|--help)
     ;;
 esac
 
-if [ $(echo $3 | grep '@' ) ]; then
-    BASTILLE_JAIL_IP=$(echo $3 | awk -F@ '{print $2}')
-    BASTILLE_JAIL_INTERFACES=$( echo $3 | awk -F@ '{print $1}')
+if echo "$3" | grep '@'; then
```

Comment:
I think we can remove this altogether. We've standardized on the final argument being the optional interface. This tried to do the same thing.